GNUmakefile

```diff
@@ -3,8 +3,9 @@ GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 WEBSITE_REPO=github.com/hashicorp/terraform-website
 
 MODULE   = $(shell env GO111MODULE=on go list -m)
-VERSION ?= $(shell git describe --tags --always --dirty --match=v* 2> /dev/null || \
+GIT_VERSION ?= $(shell git describe --tags --always --dirty --match=v* 2> /dev/null || \
 			cat $(CURDIR)/.version 2> /dev/null || echo v0)
+VERSION = $(shell echo $(GIT_VERSION) | sed 's/^v//' | sed 's/-.*//')
```

Comment:
note: changed last sed from -dirty to -.* as we can also get git versions like `v2.1.1-2-edbe3a`